faithcomesbyhearing/dbp
# FCBH-664 Get verse content on Annotations

# Description
- Added verse_text on highlights, bookmarks, and notes
- Updated documentation

## Issue Link
Original Story: [FCBH-664](https://fullstacklabs.atlassian.net/browse/FCBH-664) 
Review Story: [FCBH-796](https://fullstacklabs.atlassian.net/browse/FCBH-796) 

## How Do I QA This
- Run `http://dbp.test/open-api-v4.json` to get the new documentation
- Call `/users/{user_id}/bookmarks` and verify that now the response has `verse_text` 
- Call `/users/{user_id}/notes` and verify that now the response has `verse_text` 
- Call `/users/{user_id}/highlights` and verify that now the response has `verse_text` 

## Checklist
